admin/admin-base.php

```diff
@@ -104,7 +104,7 @@ public function init() {
 
 		// Filter admin language for users
 		// We must not call user info before WordPress defines user roles in wp-settings.php
-		add_action( 'setup_theme', array( $this, 'init_user' ) );
+		add_action( 'after_setup_theme', array( $this, 'init_user' ) );
```

Comment:
Instead of doing that we may just remove our call to `load_plugin_textdomain()` in `init_user()` as this useless for a while and rely only on `_load_textdomain_just_in_time()`.